### PR TITLE
Update cas-server version to use the forgot password feature in cas

### PR DESCRIPTION
**Pull request by candyam5522**

### CODE REVIEW

**Comment by arpost** [comment]
Miao,

Two comments:

1. Is there functionality in eureka that implemented the old forgot password functionality that needs to be deleted?
2. I try to avoid committing dependencies on a SNAPSHOT release to upstream. Once there is an Alpha release of cas that implements the new forgot password functionality, depend on that instead.

Thanks!

**Comment by candyam5522** [comment]
Answers your questions:
1. There should not be any code in eureka for old forgot password functionality, I already deleted it. But I will double check, there might be some code related to old forgot password function in user-* projects, I will look into those modules to make sure
2. Okay, got it, thanks.

File: eureka-webapp/src/main/webapp/eureka/module.js [hunk 1]
```diff
@@ -71,13 +71,21 @@
...
+        $httpProvider.defaults.headers.get['If-Modified-Since'] = 'Mon, 26 Jul 1997 05:00:00 GMT';
+        $httpProvider.defaults.headers.get['Cache-Control'] = 'no-cache, no-store, must-revalidate';
+        $httpProvider.defaults.headers.get['Pragma'] = 'no-cache';
+        
         $urlRouterProvider.otherwise('/index');
 
         $stateProvider
             .state('index', {
                 url: '/index',
+                cache: false,
```
Comment (arpost) [inline eureka-webapp/src/main/webapp/eureka/module.js]: Is this for caching the contents within angular for when a user goes back to the index page? If so, take that out, and I'll commit the other part. I was just looking for eliminating the caching that occurs when the user first navigates to Eureka.

**Comment by candyam5522** [comment]
You are right.
By default, views are cached to improve performance.
To disable it only for a single state, just add cache: false to the state definition.
So for eliminating the caching that occurs when the user first navigates to Eureka, I should not put cache:false over there.
I took it out now, please review again

**Pull request merged**